### DISCUSSION TITLE
<podcast:section>

### DISCUSSION THREAD

**Discussion by saerdnaer** [Podcastindex-org/podcast-namespace#497] [Tag Proposal]
Followup from [discussion on podcastindex.social](https://podcastindex.social/@agates/107791789803966654):

In some cases a podcast feed does not consists of individual episodes, but several parts of an episode, e.g.
- when broadcasters publish the individual stories of a magazine programme as individual items…
- when audio books are published as separate mp3 file per chapter
- whats about music albums? Are tracks/songs individual episodes or the whole album one entity?

Reusing `<podcast:episode>` to for the chapter / item number inside of that episode feels wrong, so we should probably think about an sub-entity below e.g. `section`?

Im not sure if the term `section` is really the best one... Maybe `track` or `clip`? Do you know a better term or is it just fine?

```xml
<item>
           <podcast:section name="Chapter 1">1</podcast:section>
</item>
```

Related: https://github.com/Podcastindex-org/podcast-namespace/issues/263

**Comment by daveajones** [comment]
Could we add an extra attribute to `<podcast:episode>` to indicate that it is a section instead of a new tag?  How does that feel to you?

**Comment by theDanielJLewis** [comment]
I think this would be redundant because we already have the Season tag that can do this. Plus, the episode number doesn't have to be unique: you can have an episode 5 for every season.

Yes, "season" isn't the most universal term for this, but our expansion of the season tag can do exactly what you're asking for.

**Comment by saerdnaer** [comment]
@theDanielJLewis the best generic term for a group of episodes I became aware is `grouping`, but thats an other topic than this issue as it is another hierarchy level. I can create a new issue for if you are interested in this discussion :-) 

@daveajones Something like this?

```xml
<item>
           <podcast:episode section="Part 1">233</podcast:episode>
</item>
<item>
           <podcast:episode section="Part 2">233</podcast:episode>
</item>
<item>
           <podcast:episode section="Part 3">233</podcast:episode>
</item>
```

**Comment by theDanielJLewis** [comment]
Or `group` or `series`.

But I wonder what does this do that we can't already do with the `<podcast:season>` tag and named seasons? What would make this section/group/series/whatever tag unique and _not_ overlap seasons? Or should we scrap "seasons" and switch to this new idea?

("Category" and "tag" could be other good attributes for the `<podcast:episode>` tag, but they might get confused for genres/categories and SEO tags.)
